lib/generators/wcc/menu_generator.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module Wcc
+  class MenuGenerator < Rails::Generators::Base
+    source_root File.expand_path('templates', __dir__)
+
+    def create_menus_migration
+      now = Time.now.strftime('%Y%m%d%H%M')
+      copy_file 'generated_add_menus.ts',
+        "db/migrate/#{now}01_generated_add_menus.ts"
+    end
+
+    def ensure_migration_tools_installed
+      in_root do
+        run 'npm init -y' unless File.exist?('package.json')
+        package = JSON.parse(File.read('package.json'))
+        deps = package['dependencies']
+
+        unless deps.try(:[], 'contentful-migration-cli').present?
+          run 'npm install --save watermarkchurch/migration-cli'
+        end
+      end
+    end
+
+    def ensure_wrapper_script_in_bin_dir
```

Comment:
more generic than menus, right? pull out later when it makes sense to?

Reply:
yeah I think eventually this stuff lives in the install generator.  Or - maybe all the generators include the common stuff from the install generator that just ensures that contentful migrations are set up